packages/pharos/src/components/image-card/pharos-image-card.scss

```diff
@@ -168,6 +168,23 @@ slot[name='image']::slotted(img) {
   height: auto;
 }
 
+.card__link--collection--error {
+  background-color: var(--pharos-color-marble-gray-94);
+  height: 11.25rem;
+  width: 15rem;
+  display: flex;
+  flex-direction: column;
```

Comment:
Copied most of this from the other error class, but don't like the hardcoded dimensions. Going to work on updating it to be responsive.